rasa/cli/arguments/interactive.py

```diff
@@ -40,6 +42,7 @@ def set_interactive_arguments(parser: argparse.ArgumentParser):
     add_dump_stories_param(train_arguments)
     add_force_param(train_arguments)
     add_persist_nlu_data_param(train_arguments)
+    add_server_arguments(parser)
```

Comment:
I think we should not add the other server arguments, and then not using them. I suggest just adding the `--port` argument for now

Reply:
ok.